http/service.go

```diff
@@ -78,6 +81,8 @@ func (s *Service) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		s.handleKeyRequest(w, r)
 	} else if r.URL.Path == "/join" {
 		s.handleJoin(w, r)
+	}else if r.URL.Path == "/exit" {
```

Comment:
I think you need to format your code.

Reply:
yes, maestro, thanks for your reminder, and i just find that there's no need to add exit for raft can handle it. my previous idea is to solve 'three nodes in cluster and one die' which violate the principle of raft.